pkg/diagnostics/http_monitoring.go

```diff
@@ -242,6 +273,10 @@ func (h *httpMetrics) Init(appID string, legacy bool) error {
 	} else {
 		serverTags = []tag.Key{appIDKey, httpMethodKey, httpStatusCodeKey}
 		clientTags = []tag.Key{appIDKey, httpStatusCodeKey}
+		if h.pathMatchingEnabled() {
+			serverTags = append(serverTags, httpPathKey)
+			clientTags = append(clientTags, httpPathKey, httpMethodKey)
```

Comment:
Should a non-empty slice in one direction also enable the other? Is that what is happening?

Reply:
yeah, we can split each case 👍